SPECS/rubygem-async-http/rubygem-async-http.spec

```diff
@@ -1,3 +1,6 @@
+# Disabled debuginfo package as the autogenerated 'debugfiles.list' file is empty.
```

Comment:
Should we bump the Release tag, because we did when we re-enabled for `ffi` package?

Reply:
I decided to not bump it, since this never built. It didn't want to make it seem like we ever had a working package with debuginfo.